content/2019-munich/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Overview
+---
+
+## Overview
+
+PromCon EU 2019 is the fourth conference dedicated to the
+[Prometheus monitoring system](https://prometheus.io/). It will take place
+2019-11-07 & 2019-11-08 (Thu & Fri)
+at Google Munich as a single-track event with space for 220 attendees.
+
+PromCon aims to connect Prometheus users and developers from around the world in
+order to exchange knowledge, best practices, and experience gained around using
+Prometheus. We also want to collaborate to build a community and grow
+professional connections around systems and service monitoring.
+
+Get an impression of PromCon 2017:
+
+<%= youtube_player("4Pr-z8-r1eo") %>
+
+and 2018:
+
+<%= youtube_player(""GW3MzPecU94) %>
+
+## Registration
+
+Registration for PromCon 2019 is closed.
```

Comment:
"is closed" -> "will open soon" (otherwise it sounds like it was already open and is now closed)